compose/material/material/src/desktopTest/kotlin/androidx/compose/material/DesktopAlertDialogTest.kt

```diff
@@ -115,36 +115,6 @@ class DesktopAlertDialogTest {
         }
     }
 
-    // https://github.com/JetBrains/compose-multiplatform/issues/2857
-    @OptIn(ExperimentalMaterialApi::class)
-    @Test
-    fun `shadow drawn at content bounds`() {
```

Comment:
Fails after clipping, but not relevant anymore - there is no shadow in common implementation now